DevIncubator-Autopark/Program.cs

```diff
@@ -1 +1,76 @@
-﻿Console.WriteLine("Hello, World!");
+﻿using System.Collections.Generic;
+
+namespace DevIncubatorAutopark
+{
+    class Program
+    {
+        public static void Main()
+        {
+            //1
+            VehicleType[] vehicleTypes = new VehicleType[]
+            {
+               new VehicleType("Bus", 1.2f),
+               new VehicleType("Car"),
+               new VehicleType("Rink", 1.5f),
+               new VehicleType("Tractor", 1.2f)
+            };
+
+            //2
+            foreach (var vehicleType in vehicleTypes)
+            {
+                vehicleType.Display();
+            }
+
+            //3
+            vehicleTypes[^1].TaxCoefficient = 1.3f;
+
+            //4
+            Console.WriteLine($"Max TaxCoefficient = {GetMaxTaxCoefficient(vehicleTypes)}");
+
+            //5
+            Console.WriteLine($"Average TaxCoefficient = {GetAverageTaxCoefficient(vehicleTypes)}");
+
+            //6
+            float maxTaxCoefficient = 0;
```

Comment:
Usually for float-pointed numbers is using decimal or double. Pls don't use float because it has bad possibility to upcasting and not huge range of numbers.

Reply:
fixed